spec/support/authenticated_session_helper.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+module AuthenticatedSessionHelper
+  def use_authenticated_current_user(options = {})
+    klass = options[:klass] || ApplicationController
```

Comment:
See I think there should be. For secure messaging we have too way too much complexity to do request specs for everything. It would require testing filters and sorting at a very granular level. As such were treating request specs as a test for route and controller with proper handling of params and were counting on our unit level tests for things like validations, sorting; filtering to work. I think this, being an inherited method falls into the same category. Test it once, s tr in it elsewhere.